docs/plugins/gatsby-theme-flow/src/components/flow-status/hooks.js

```diff
@@ -48,6 +49,39 @@ export function useBreakingChangesPosts() {
   return posts || [];
 }
 
+export function useMainnetSporkPosts() {
+  const [posts, setPosts] = useState([]);
+
+  useSWR(DISCOURSE_MAINNET_SPORK_API_URL, fetchDiscourseChanges, {
+    refreshInterval: 100000,
+    onSuccess(data) {
+      const {
+        topic_list: { topics }
+      } = data;
+
+      const sorted = topics
+          // Removes the "About this category" post
+          .filter((post) => post.id !== 2543)
+          .map((post) => {
+            const date = moment(new Date(post.created_at)).fromNow();
+            post.__formatted_date = date;
+            return post;
+          })
+          // Couldn't find any sorting options on the API side
+          .sort((a, b) => {
+            return new Date(a.created_at).getTime() >
+            new Date(b.created_at).getTime()
+                ? -1
+                : 1;
+          })
```

Comment:
I tried doing `.first()` after this but it didn't work :( ..it results in an sorted being empty.
Ideally, the new section should only display the next upcoming spork and not the past ones but I am not sure how to get that to work - appreciate your inputs